src/Runner.Worker/ActionManifestManager.cs

```diff
@@ -57,7 +57,7 @@ public override void Initialize(IHostContext hostContext)
 
         public ActionDefinitionData Load(IExecutionContext executionContext, string manifestFile)
         {
-            var templateContext = CreateContext(executionContext);
+            var templateContext = CreateTemplateContext(executionContext);
```

Comment:
sorry about the diff for this file, i renamed `context` local variable everywhere to `templateContext` for future readability (distinguish from executionContext)